### ISSUE TITLE
The label of epo:addressID

### ISSUE THREAD

**Issue by kvistgaard** [OP-TED/ted-rdf-mapping#201]
It may need fixing.
Now it is `Address i d @en`

**Comment by csnyulas** [comment]
This sounds like an ePO modelling problem. Or was this fond in a mapping file, or the result created by some mapping?

ePO issues should be created here: https://github.com/OP-TED/ePO/issues/

**Comment by kvistgaard** [comment]
Indeed. It was created here my mistake. It needs to be [transferred](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) to the right repo but I don't have the privilege to do so.
